.eslintrc.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
-	extends: ["eslint:recommended"],
+	extends: ["eslint:recommended", "plugin:prettier/recommended"],
```

Comment:
No need to add `prettier/prettier` in rules as the plugin does that for us

Reply:
okay I think this is good